app/views/sessions/new.html.erb

```diff
@@ -6,34 +6,57 @@
 
   <h2 class='text-3xl font-semibold'>Welcome back</h2>
 
-  <span class='text-sm text-center'>
-    Don't have an account? <%= link_to "Sign up", register_path, class: "underline" %>
-  </span>
+  <% if Feature.authenticate_with_password? %>
+    <span class='text-sm text-center'>
+      Don't have an account? <%= link_to "Sign up", register_path, class: "underline" %>
+    </span>
+  <% end %>
```

Comment:
In addition to hiding this link, somneone could visit this /users/new route directly so we should add a redirect in users_controller on the new route. Do we also need one on the create route? Probably would be a good idea so maybe a before_filter that applies to them both.